### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by shadowas-py**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,39 +1,20 @@
...
-          >
-            Vite Docs
-          </a>
-        </p>
-      </header>
-    </div>
+    <BrowserRouter>
+      <div className="App">
+        <Routes>
+          <Route path="/register" element={<h1>REGISTER PAGE</h1>} />
```
Comment (ike-tom) [inline src/App.jsx]: register page will open in the modal so I'm not sure we need additional route to this page. The same for login page. After clicking sign in the modal will apear. Inside this modal you will have a switch to change modal between login modal and register modal.

File: src/App.jsx [hunk 2]
```diff
@@ -1,39 +1,20 @@
...
+          <Route path="/" element={<h1>INDEX PAGE</h1>} />
+          <Route path="/login" element={<h1>LOGIN PAGE</h1>} />
```
Comment (ike-tom) [inline src/App.jsx]: same as comment for register page above

File: src/App.jsx [hunk 3]
```diff
@@ -1,39 +1,20 @@
...
+          <Route path="/user" element={<h1>USER PAGE</h1>} />
+          <Route path="/addhouse" element={<h1>ADD_HOUSE PAGE</h1>} />
```
Comment (ike-tom) [inline src/App.jsx:12]: you can add route for favourites

**Review by ike-tom — CHANGES_REQUESTED** [review]
please make corrections. Thank you :)

**Review by ike-tom — APPROVED** [review]

**Review by FilipHalon — APPROVED** [review]

**Pull request merged**